README.md

```diff
@@ -19,6 +19,25 @@ This repo contains the code for generating two Docker images for Swift:
 7. Updated Dockerfiles per guidelines in [Best practices for writing Dockerfiles](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/).
 8. Added Support for amd64 and s390x architectures of Xenial Linux Ubuntu 16.04.
 
+
+
+# Quick Start
+
+The easiest way to get started is by using the Kitura command line tools to generate the docker files needed, and copy them over to your project:
+
+1. Create a new directory, move into it and run `kitura init`. This will generate a project along with two files named `Dockerfile` and`Dockerfile-tools`. 
+2. Copy `Dockerfile` and `Dockerfile-tools` into your projects root directory (same one that contains the Sources and Tests folders and your `Package.swift` file).
```

Comment:
Copy the `Dockerfile` and `Dockerfile-tools` files into your project's root directory (this is the directory that contains your `Package.swift` file and the Sources and Tests folders).

Reply:
✅